components/rmrk/Gallery/Search/SearchBarCollection.vue

```diff
@@ -148,3 +148,14 @@ export default class SearchBar extends Vue {
   opacity: 0;
 }
 </style>
+
+<style lang="scss">
+.field-group-container {
+  .is-grouped-multiline {
+    flex-wrap: initial!important;
+    @media screen and (max-width: 768px) {
+      flex-wrap: wrap!important;;
+    }
+  }
+}
```

Comment:
Oh my, what does this fix?

Reply:
@roiLeo due to the `is-group-multiline` class from buefy, it was being applied for desktop also, so` flex-wrap` was messing things on smaller desktop screens, so I applied flex-wrap only on mobile screens and scoped it through the parent class `field-group-container`